Modix/Modules/InfractionModule.cs

```diff
@@ -67,39 +70,51 @@ public InfractionModule(IModerationService moderationService)
             var replyBuilder = new StringBuilder();
             foreach (var line in tableText.Split("\r\n"))
             {
-                if((replyBuilder.Length + line.Length) > 1998)
+                if ((replyBuilder.Length + line.Length) > 1998)
                 {
                     await ReplyAsync(Format.Code(replyBuilder.ToString()));
                     replyBuilder.Clear();
                 }
                 replyBuilder.AppendLine(line);
             }
 
-            if(replyBuilder.Length > 0)
+            if (replyBuilder.Length > 0)
                 await ReplyAsync(Format.Code(replyBuilder.ToString()));
         }
-        
+
+        [Command("search")]
+        [Summary("Display all infractions for a user, that haven't been deleted.")]
+        public async Task Search(
+            [Summary("The user whose infractions are to be displayed.")]
+                IGuildUser subject)
+        {
+            await Search(subject.Id);
+        }
+
         [Command("search embed")]
         [Summary("Display all infractions for a user, that haven't been deleted.")]
+        [Priority(10)]
         public async Task SearchEmbed(
             [Summary("The user whose infractions are to be displayed.")]
-            IGuildUser subject)
+            ulong subjectId)
         {
             var requestor = Context.User.Mention;
-            
+            var subject = await UserService.GetGuildUserSummaryAsync(Context.Guild.Id, subjectId);
+
             var infractions = await ModerationService.SearchInfractionsAsync(
                 new InfractionSearchCriteria
                 {
-                    GuildId = subject.GuildId,
-                    SubjectId = subject.Id,
-                    IsDeleted = false
+                    GuildId = Context.Guild.Id,
+                    SubjectId = subjectId,
+                    IsDeleted = false,
+                    IsRescinded = false
```

Comment:
Why add `IsRescinded` here? Last I recall, we explicitly wanted to not filter out rescinded infractions. Like, if someone was tempmuted, for an hour, once that hour is up, Modix automatically rescinds the infraction, so it'll forevermore not show up in this command, when it's probably quite relevant to know if someone did something worthy of a tempmute.

Reply:
What I probably should do is modify the command embed here - right now, there's no indicator of if an infraction is active or rescinded, so I didn't want to cause confusion. I'll do that instead.

Reply:
If it's in the interest of saving space, I suppose I'm okay with leaving them out.

I'm tempted to just say "forget displaying infractions in Discord, just link to the web and be done with it."